refinery/tool_manager/serializers.py

```diff
@@ -13,10 +13,30 @@ class Meta:
 
 
 class ParameterSerializer(serializers.ModelSerializer):
+    galaxy_workflow_step = serializers.SerializerMethodField(
+        method_name="_get_galaxy_workflow_step", required=False,
```

Comment:
For readability, suggestion:
galaxy_workflow_step = serializers.SerializerMethodField(
       method_name="_get_galaxy_workflow_step", 
       required=False,
       allow_null=False
)